src/OpenTelemetry.Api/Context/RuntimeContext.cs

```diff
@@ -23,7 +23,7 @@ namespace OpenTelemetry.Context
     /// <summary>
     /// Generic runtime context management API.
     /// </summary>
-    public sealed class RuntimeContext
+    public static class RuntimeContext

```

Comment:
why this change?

Reply:
Since it just have static methods, it was expecting to be static class as well. I can revert if needed.

Reply:
Seems static fits better in this case.